src/snowflake/sqlalchemy/util.py

```diff
@@ -20,27 +20,13 @@ def _url(**db_parameters):
         raise exc.ArgumentError("account parameter must be specified.")
 
     if "host" in db_parameters:
-        ret = "snowflake://{user}:{password}@{host}:{port}/".format(
-            user=db_parameters.get("user", ""),
-            password=_rfc_1738_quote(db_parameters.get("password", "")),
-            host=db_parameters["host"],
-            port=db_parameters["port"] if "port" in db_parameters else 443,
-        )
+        ret = f"snowflake://{db_parameters.get('user', '')}:{_rfc_1738_quote(db_parameters.get('password', ''))}@{db_parameters['host']}:{db_parameters['port'] if 'port' in db_parameters else 443}/"
         specified_parameters += ["user", "password", "host", "port"]
     elif "region" not in db_parameters:
-        ret = "snowflake://{user}:{password}@{account}/".format(
-            account=db_parameters["account"],
-            user=db_parameters.get("user", ""),
-            password=_rfc_1738_quote(db_parameters.get("password", "")),
-        )
+        ret = f"snowflake://{db_parameters.get('user', '')}:{_rfc_1738_quote(db_parameters.get('password', ''))}@{db_parameters['account']}/"
         specified_parameters += ["user", "password", "account"]
     else:
-        ret = "snowflake://{user}:{password}@{account}.{region}/".format(
-            account=db_parameters["account"],
-            user=db_parameters.get("user", ""),
-            password=_rfc_1738_quote(db_parameters.get("password", "")),
-            region=db_parameters["region"],
-        )
+        ret = f"snowflake://{db_parameters.get('user', '')}:{_rfc_1738_quote(db_parameters.get('password', ''))}@{db_parameters['account']}.{db_parameters['region']}/"
```

Comment:
We could add port to this too

Reply:
may I ask to which part we could add the port? is it after the region?
I checked the samples looks like all port come with host

Reply:
okay, I tried to add it but found it might become a behavior breaking change for the `URL` method (or  a bug fix depending on how we look at it).

- previously if we do `URL(account="testaccount", user="admin", password="test", database="testdb", port=8080)`, we get string `snowflake://admin:test@testaccount/testdb?port=8080` back.
- if we made the change, it would be `snowflake://admin:test@testaccount:8080/testdb`

both style could get the `port` populated correctly, but the string itself changes.. so I'm a bit reluctant to introduce the change unless users ask for it.

thoughts?